src/Symfony/Component/Clock/TimeInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+interface TimeInterface
+{
+    public function now(): \DateTimeImmutable;
```

Comment:
WDYT about `now(?DateTimeZone $tz  = null)` to bypass TZ conversion and being more consise also

```
$clock->now($user->getTz())
$clock->now()->setTz($user->getTz())
```

Reply:
i wouldnt mind an accessor for default TZ either :)

Reply:
The accessor and the setter already exists: `->now()->get/setTimezone()`.
If you mean changing the default TZ of the clock service, we could consider adding `->withTimeZone()`.
I'd just prefer postponing the discussion on this idea to a follow up PR.

Reply:
i meant "to bypass TZ conversion and being more consise also"

so similar like we do `time()` over `now()->getTimestamp()`, isnt it?

i actually prefer the shorthands now, esp. `now($tz)`

Reply:
i should have measured :} i see some gain, but i think the numbers are negligible yes

i was mostly pursuing now($tz) for more consise API, it's not a blocker